modules/meta/arrow-annotations/src/main/java/arrow/extension.kt

```diff
@@ -4,3 +4,9 @@ package arrow
 @Target(AnnotationTarget.CLASS)
 @MustBeDocumented
 annotation class extension
+
+val given: Nothing
+  get() = TODO("Should have been replaced by Arrow Meta Compiler Plugin provided by [apply plugin: 'arrow.meta.plugin.gradle']")
```

Comment:
Now that we know how a user should include the plugin we should replace this TODO message for a better one that tells them in which place they need to add the plugin

Reply:
Does not need to happen in this PR but this is what the user will see if the plugin is not activated and they attempt to run some code that depends on `given`